docs/development_guide/README.md

```diff
@@ -75,6 +75,45 @@ Run `go get -d ./...` to download all required Go packages.
 
 Also the project requires Docker. See how to [install Docker](https://docs.docker.com/engine/install/).
 
+## Getting started
+
+This will give you the basic commands to get started with development.  The rest of this file goes into depth on the make commands, linting, and testing.  It also goes over how to customize the deployment through the `.env` file to deploy optional components or deploy to different environments. 
+
+```bash
+# use default setup
+cp .env.example .env
+source .env
+
+# create a local kind cluster
+make kind-up
+
+# build osm controller components and cli (includes changes to helm chart)
+make build-osm-all
+
+# deploy osm
+./bin/osm install --set=osm.image.registry="$CTR_REGISTRY" --set=osm.image.pullPolicy=Always --verbose 
```

Comment:
Should we add "CTR_TAG" here as well, in the case users want to test a version other than latest-main?

Reply:
I don't think its necessary for basic command due to the defuaults but its a good idea to add a note here that you could set TAG or fields as needed using CRT_TAG as an example.  Wdyt?

Reply:
I think that makes sense

Reply:
added